src/service/poc_mobile.proto

```diff
@@ -106,22 +86,31 @@ enum share_validity {
   share_validity_not_operational = 5;
 }
 
-message moving_avg {
+message speedtest_avg {
   bytes pub_key = 1;
-  /// Refers to the number of speed_shares used to calculate the average
-  uint64 window_size = 2;
```

Comment:
Any particular reason why window_size was removed? We might use it in Explorer to tell user on how many datapoints their average is calculated.

Reply:
the window size is the number of speedtests attached to the average, so we removed this since it is duplicated information.

Reply:
Understood, thanks!